spec/rubocop/cop/layout/empty_lines_around_attribute_accessor_spec.rb

```diff
@@ -47,6 +47,10 @@ def do_something
     RUBY
   end
 
+  it 'accepts code that where the attr_accessor is the last line' do
+    expect_no_offenses("attr_accessor :foo")
```

Comment:
CI is failing. Can you update this one?

```suggestion
    expect_no_offenses('attr_accessor :foo')
```

Reply:
🤦 Too much time in the Chef codebase